src/runtime/config/configuration-qemu-coco-dev.toml.in

```diff
@@ -713,3 +713,7 @@ create_container_timeout = @DEFCREATECONTAINERTIMEOUT@
 # to the hypervisor.
 # (default: /run/kata-containers/dans)
 dan_conf = "@DEFDANCONF@"
+
+# Enforce guest pull. This instructs the runtime to communicate to the agent via annotations that
+# the container image should be pulled in the guest, without using an external external snapshotter.
```

Comment:
Double external (here and in the other config files)

Reply:
fixed